src/formViews/Welcome.js

```diff
@@ -13,26 +13,32 @@ function Welcome() {
     <Fragment>
       <img alt="logo" src={logo} className="image-logo" />
       <div className="welcome-row-1">
-        <img alt="timer" src={imageTimer} className="image-timer" />
-        <p className="primary-text">Sign up to vote in under 5 mins!</p>
-        <p className="secondary-text">No printing, no posting, no fuss!</p>
+        <p className="primary-text">Sign up to vote</p>
+        <p className="primary-subtext">in 5 minutes</p>
+        <div className="bullets">
+          <ul className="bullets-list">
+            <li className="bullets-listitem"><span>UK & EU Citizens</span></li>
+            <li className="bullets-listitem"><ModalForms /></li>
+            <li className="bullets-listitem"><span>Guideline compliant</span></li>
+          </ul>
+        </div>
       </div>
-      <Link className="Button" to="/typeOfVoter/">Start Now</Link>
+      <Link className="Button welcome" to="/typeOfVoter/">Start Now</Link>
       <div className="welcome-row-2">
         <p className="welcome-row-2-element">
           <img alt="filled out form" src={imageFillForm} className="image-row-2" />
-          <span className="secondary-text">Fill in your details</span>
+          <span className="secondary-text">Enter your details</span>
```

Comment:
I don't know if prefer `Enter` to `Fill in` has this been agreed to in the group?

Reply:
👍